.github/workflows/python-app.yml

```diff
@@ -31,8 +31,8 @@ jobs:
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+        # The GitHub editor is 127 chars wide but we prefer 79 for PEP8.
+        flake8 . --count --max-complexity=10 --max-line-length=79 --statistics
```

Comment:
that  duplication of flake call was because the first break the build and the second don't. Now both break, so why don't use only the greater, in that case, the last?